Userland/Libraries/LibJS/Bytecode/Interpreter.cpp

```diff
@@ -876,6 +876,24 @@ inline void fast_typed_array_set_element(TypedArrayBase& typed_array, u32 index,
     *slot = value;
 }
 
+Completion throw_null_or_undefined_property_get(VM& vm, Value base_value, Optional<IdentifierTableIndex> base_identifier, IdentifierTableIndex property_identifier, Executable const& executable)
```

Comment:
this (and the other function below) are missing a `static`

Reply:
will be fixed in #626, incidentally